routes/scan.js

```diff
@@ -13,6 +13,6 @@ const urlEncodedParser = bodyParser.urlencoded({ extended: false });
 router.post("/", urlEncodedParser, asyncMiddleware(post));
 router.get("/", get);
 router.get("/full_report", getFullReport);
-router.get("/latest_breaches", getLatestBreaches);
+router.get("/user_dashboard", getUserDashboard);
```

Comment:
Let's consider if we can just show the user dashboard at `/` url instead of redirecting signed-in users to `/user_dashboard`. @sandysage - should we file and issue to consider this? Or just discuss and decide in a meeting or something?